conans/client/conan_api.py

```diff
@@ -601,6 +601,11 @@ def config_install(self, path_or_url, verify_ssl, config_type=None, args=None,
                                      args=args,
                                      source_folder=source_folder, target_folder=target_folder)
 
+    @api_method
+    def config_home(self):
+        self.app.out.info(self.cache_folder)
```

Comment:
Actually it is more correct that the api returns the value, and the Command class is the one printing the output.

Reply:
so we need to revisit `config get`, because it follows the same steps.

Reply:
indeed I agree with you, but should I change `config get` ?

Reply:
No, you can leave it, there will be also many other things that would need to be fixed exactly the same. Just wanted to say it to outline future direction of the conan_api.

Reply:
okay, I gonna update config home only then

Reply:
done